lib/api/1.1/northbound/profiles.js

```diff
@@ -2,9 +2,9 @@
 
 'use strict';
 
-var di = require('di'),
-    express = require('express'),
-    parser = require('body-parser');
+var di = require('di');
+var express = require('express');
+var ambiguousParser = require('./../../../fittings/ambiguous_text_parser');
```

Comment:
I'm not too excited about the deep relative path here, partly because it circumvents how we do dependency management. But I know we're also going in a direction with the 2.0 API that can't use DI because of swagger. And is fittings the right place for this? @brianparry do you have any thoughts here?